package.json

```diff
@@ -9,7 +9,8 @@
     "fmt": "prettier --write .",
     "prepare": "husky",
     "test": "mocha 'src/**/*.test.ts'",
-    "coverage": "c8 yarn test"
+    "coverage": "c8 yarn test",
+    "release": "commit-and-tag-version"
```

Comment:
This without an instruction to skip tagging will result in tagging the commit in the release branch, where we usually prefer to wait on tagging until it hits the main branch in case there are any changelog adjustments/etc that have to happen within the release branch.